requirements/REQUIREMENTS.md

```diff
@@ -0,0 +1,27 @@
+##Предполагаемая ЦА
+Пользователи мессенджеров, соцсетей и иных сервисов, которых не удовлетворяет функционал данных сервисов в области опросов.
+
+##Разбиение ЦА
```

Comment:
That is not classes of users... :) That are basically sources of possible traffic.